File: PROJECT/Models/DTOs/Internal/Actions/ActionReportDTO.cs
namespace Models.DTOs.Internal.Actions
{
    public class ActionReportDTO
    {
        public int UserId { get; set; }
        public string UserFirstName { get; set; }
        public string UserLastName { get; set; }
        public decimal Paycheck {get; set; }
        public List<ActionRecordDTO> Actions { get; set;}
    }
}

File: PROJECT/FRONTEND/Common/DTOs/ActionReportDTO.ts


export class ActionReportDTO { 
    constructor(obj:Partial<ActionReportDTO>) {
        Object.assign(this,obj);
    }
    
    public userId: number;
    public userFirstName: string;
    public userLastName: string;
    public paycheck: number;
    public actions: ActionRecordDTO[];
}
